phpunit

```diff
@@ -51,11 +54,13 @@ if ($phpIniMatrix) {
 
     $phpDir = dirname(`where.exe php`);
 
-    $newCmd = '(SET X=0';
+    $newCmd = 'cmd /v:on /d /c "(SET X=0';
```

Comment:
are you sure that windows flags are case insensitive ? they are uppercase in the Process component